diff -Naur orig/tensorflow/tensorflow/contrib/mpi_collectives/BUILD tensorflow/tensorflow/contrib/mpi_collectives/BUILD
--- orig/tensorflow/tensorflow/contrib/mpi_collectives/BUILD	2020-03-01 21:35:56.653455174 +0000
+++ tensorflow/tensorflow/contrib/mpi_collectives/BUILD	2020-02-26 22:21:50.398949821 +0000
@@ -53,6 +53,7 @@
         ":mpi_defines",
         ":mpi_message_proto_cc",
         "//third_party/mpi",
+	"//tensorflow/stream_executor",
     ],
 )
 
diff -Naur orig/tensorflow/tensorflow/core/platform/platform.h tensorflow/tensorflow/core/platform/platform.h
--- orig/tensorflow/tensorflow/core/platform/platform.h	2020-03-01 21:35:57.929494015 +0000
+++ tensorflow/tensorflow/core/platform/platform.h	2020-02-26 22:21:01.017621562 +0000
@@ -45,9 +45,9 @@
 
 // Require an outside macro to tell us if we're building for Raspberry Pi or
 // another ARM device that's not a mobile platform.
-#if !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
-#define IS_MOBILE_PLATFORM
-#endif  // !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
+//#if !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
+//#define IS_MOBILE_PLATFORM
+//#endif  // !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
 
 #else
 // If no platform specified, use:
diff -Naur orig/tensorflow/tensorflow/tensorflow.bzl tensorflow/tensorflow/tensorflow.bzl
--- orig/tensorflow/tensorflow/tensorflow.bzl	2020-03-01 21:36:01.117591057 +0000
+++ tensorflow/tensorflow/tensorflow.bzl	2020-02-26 22:22:28.095965476 +0000
@@ -1798,7 +1798,7 @@
         name = name + "_check_deps",
         disallowed_deps = [
             clean_dep("//tensorflow/core:framework"),
-            clean_dep("//tensorflow/core:lib"),
+            # clean_dep("//tensorflow/core:lib"),
         ],
         deps = deps + if_cuda_is_configured_compat(cuda_deps) + if_rocm_is_configured(rocm_deps),
     )
diff -Naur orig/tensorflow/third_party/repo.bzl tensorflow/third_party/repo.bzl
--- orig/tensorflow/third_party/repo.bzl	2020-03-01 21:36:02.361628923 +0000
+++ tensorflow/third_party/repo.bzl	2020-02-27 08:12:21.139361042 +0000
@@ -51,7 +51,7 @@
     if result.return_code != 0:
         fail(("Non-zero return code({1}) when executing '{0}':\n" + "Stdout: {2}\n" +
               "Stderr: {3}").format(
-            " ".join(cmd_and_args),
+            " ".join([str(x) for x in cmd_and_args]),
             result.return_code,
             result.stdout,
             result.stderr,
